qa/rolling-upgrade/src/javaRestTest/java/org/elasticsearch/upgrades/HealthNodeUpgradeIT.java

```diff
@@ -26,6 +29,12 @@ public HealthNodeUpgradeIT(@Name("upgradedNodes") int upgradedNodes) {
         super(upgradedNodes);
     }
 
+    @Before
+    public void updatePollInterval() throws IOException {
+        // We need Data Stream Lifecycle to run at least once to have a "green" status.
+        updateClusterSettings(client(), Settings.builder().put("data_streams.lifecycle.poll_interval", "5s").build());
```

Comment:
After I added this fix, I realized that this PR will cause the health API to return `unknown` after cluster startup until Data Stream Lifecycle has run at least once. The default interval is `5m`, which is quite a while for the health API to report `unknown` IMO. It probably makes sense to discuss this briefly before we proceed.